cadence/src/builder.rs

```diff
@@ -365,6 +365,40 @@ where
         self
     }
 
+    /// Add tags to this metric.
+    ///
+    /// # Example
```

Comment:
I think you'll have to remove this example now that this method isn't public.

Reply:
Oops, thanks. Forgot to run `cargo test --doc` after changing that. Removed!

I think this is good to go now, thanks again.